test/astronomy/large_semi_axis_of_orbit.py

```diff
@@ -0,0 +1,40 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from symplyphysics import (assert_equal, units, Quantity, errors,)
+from symplyphysics.laws.astronomy import large_semi_axis_of_orbit as axis_law
+
+# Description
+## The mass of the sun is 1.989e30 kilograms, and the orbital speed of the Earth is 29.8 kilometers per second.
+## Then the semi-major axis of the orbit is equal to 149.5e9 meters.
+## https://uchet-jkh.ru/i/kak-naiti-bolsuyu-poluos-orbity/
```

Comment:
This link has this law reversed, yet the same result as yours. How is it possible?

Reply:
Someone following the link might be confused. Add some comments about missing `-1` power.